city_scrapers/spiders/chi_ssa_20.py

```diff
@@ -0,0 +1,83 @@
+import re
+from city_scrapers_core.constants import NOT_CLASSIFIED
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa20Spider(CityScrapersSpider):
+    name = "chi_ssa_20"
+    agency = "Chicago Special Service Area #20 South Western Avenue"
+    timezone = "America/Chicago"
+    start_urls = ["https://www.mpbhba.org/business-resources/"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+        """
+
+
+        for h3 in response.xpath("//h3"):
+           entry_str = h3.xpath("./text()").extract_first()
+           if entry_str and "ssa meetings" in entry_str.lower():
```

Comment:
This is looking right to me so far! You could also remove extra whitespace with something like

```python
entry_str = re.sub(r"\s+", " ", entry_str)
```

Reply:
Thank you @pjsier , I can indeed work with that - but the resulting issue is that I can no longer use something like `following-siblinng` in order to traverse other items such as `<p`> that are related to my `<h3>`. Would happen to have an idea as to how to do that ?

Reply:
I'm currently trying to create a dict (or a tuple) from the resulting `h3` and see if I can isolate the meeting dates related to `ssa 20` that way. 
Then, pass those strings for date manipulation onto `parse_start`_.